sound/soc/sof/ipc4-topology.c

```diff
@@ -1374,6 +1377,9 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 			return ret;
 		}
 
+		dev_dbg(scomp->dev, "Got %d ms and %d for fast_mode flag\n",
+			tuples.deep_buffer_dma_ms, tuples.fast_mode);
```

Comment:
I think this print is not valid. deep_buffer_dma_ms is not dependent on fast mode, these are sort of independent parameters.

Reply:
But if you want to print these then, the deep buffer print is a separate one:
```c
if (deep_buffer_dma_ms)
	dev_dbg(scomp->dev, "deep buffer for %s: %u ms\n", swidget->widget->name, deep_buffer_dma_ms);
```
then along the same lines print only if the fast mode is selected for the given copier?

Reply:
Well, its probably better to get rid of the print all together. I'll just add a simple debug print about setting the fast_mode bit, to have some clear indication its there, while we are still trying to get the all the pieces to work together.